spec/models/maths_and_physics/eligibility_spec.rb

```diff
@@ -52,6 +52,12 @@
     end
   end
 
+  describe "#award_amount" do
+    it "returns the £2,000 amount that Maths & Physics claimants are eligible for" do
+      expect(MathsAndPhysics::Eligibility.new.award_amount).to eq(BigDecimal("2000"))
```

Comment:
Any reason why this `BigDecimal` doesn't have the decimal places, whereas the code does?

Reply:
So the value returned by `BigDecimal("2000")` is equivalent to the value returned by `BigDecimal("2000.00")` (which is also equivalent to the value returned by `BigDecimal("2000.00000000")`) because the class just converts the string into its decimal equivalent and that's what we're comparing here. I suppose it is a little inconsistency in the way the value is represented in the test vs the actual code, but I don't feel it has a big enough material impact to warrant a fixup/CI-rerun. What you reckon?

Reply:
Yeah, I figured they were equivalent, it just stuck out a bit. Fine for it to go in as is 👍

Reply:
cool, just need an approval on the review